test/helpers/verification.js

```diff
@@ -52,3 +53,7 @@ export async function curveInEther(money, eurEtherRatio) {
 export function ethToEur(ether, eurEtherRatio = etherToWei(218.1192809)) {
```

Comment:
All test helpers, including `verification.js` need overhaul. it is pointless to just fix this issue. it would require to move `eurToEth` to `deploy.js` where deployment config values are available so it is substantial amount of work and I doubt it will be final structure when tests are done correctly. In short: can we merge solidity and do testing improvements in separate PR?